compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/scene/ComposeScenePointer.skiko.kt

```diff
@@ -151,21 +152,41 @@ internal fun PointerInputEvent(
 /**
  * The result of processing a pointer event.
  *
- * @property anyMovementConsumed Indicates whether any pointer movement was consumed during event
- * processing.
+ * @property value is a bitmask indicating the properties of the result.
+ * Use [anyMovementConsumed] and [anyChangeConsumed] to read those properties.
+ *
+ * The value is the same as in [androidx.compose.ui.input.pointer.ProcessResult],
+ * but we can't just reuse it because it's internal.
  */
 @InternalComposeUiApi
 @JvmInline
-value class PointerEventResult(val anyMovementConsumed: Boolean)
+value class PointerEventResult(val value: Int) {
```

Comment:
Let's keep `value` as internal format?

```suggestion
value class PointerEventResult internal constructor(private val value: Int) {
```

Reply:
yes, makes total sense. Fixed

Reply:
`val value` is still public

Reply:
missed that. thanks! fixed